spk/homeassistant/src/installer.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/sh
+
+# Package
+PACKAGE="homeassistant"
+DNAME="Home Assistant"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+SSS="/var/packages/${PACKAGE}/scripts/start-stop-status"
+PYTHON_DIR="/usr/local/python3"
+PATH="${INSTALL_DIR}/env/bin:${PYTHON_DIR}/bin:${PATH}"
+USER="homeassistant"
+GROUP="users"
```

Comment:
Having a UI would be cool, but at the moment I don't have time to pick up that extra thing (getting this basic version to work already takes quite much of my time).

At the moment all configuration is done through configuration files (scripts, setting up groups, etc) and writing a wizard for it will take lots of time, while I think that eventually the configuration will also be done through the web interface.